views/best-practices.html

```diff
@@ -1,31 +1,15 @@
 {{<partials-layout}}
-  {{$menu}}
-    <main id="content" class="main" role="main">
-      <ul class="nav-menu">
-        <li><a href="/">What we do</a></li>
-        <li><a href="/hosting platform">Hosting Platform</a></li>
-        <li><a href="/teams">Teams</a></li>
-        <li><a href="/engagement">Engagement</a></li>
-        <li>Best practices</li>
-        <li><a href="/roadmap">Road map</a></li>
-        <li><a href="/resources">Resources</a></li>
-      </ul>
-  {{/menu}}
-  {{$content}}
-  <div class="grid-row">
-    <div class="column-two-thirds">
-      <section class="intro">
-        <h1>Best Practices</h1>
-        <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Mauris in fermentum lacus, at sollicitudin dolor.
-          Quisque vel dui eget velit varius tempor. Vestibulum iaculis magna mauris.
-          Cum sociis natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus.
-          Etiam ut eros eu purus malesuada rhoncus at ac dui. Integer bibendum dapibus molestie.
-          Praesent sodales ipsum tortor, vulputate pretium metus suscipit at.
-          Donec accumsan tellus vitae dapibus tincidunt. In varius quam justo.
-        </p>
-      </section>
-    </div>
-  </div>
-  </main>
-  {{/content}}
+{{$practice-active}}active{{/practice-active}}
+{{$content}}
+  <section class="intro">
```

Comment:
hard to judge based on the lorem ipsum.
the page should only have one `H1` and that should be the page title, so not sure it makes sense to put that in a `<section>`
also not sure why you ditched using `<main>`